app/gui2/src/components/GraphEditor/widgets/WidgetText.vue

```diff
@@ -99,7 +99,6 @@ export const widgetDefinition = defineWidget(
       ref="input"
       v-model="editedContents"
       autoSelect
-      @pointerdown.stop
```

Comment:
Why is this removed? The pointerdown are actually handled by input (pointerdown, not click sets the text cursor), thence it should be also stopped here.